.github/CODEOWNERS

```diff
@@ -144,8 +144,9 @@
 /StarVMC/StarBASE                                  @klendathu2k
 /StarVMC/StarGeometry                              @klendathu2k
 /StarVMC/StarVMCApplication                        @yfisyak
-/StarVMC/xgeometry                                 @klendathu2k       @yfisyak
+/StarVMC/xgeometry                                 @klendathu2k       
```

Comment:
```suggestion
/StarVMC/xgeometry                                 @klendathu2k       @yfisyak
```

Reply:
No. Software coordinators do not own the full STAR geometry. (On request will add them to ownership of their individual subsystems).